plugins/woocommerce-blocks/assets/js/data/cart/test/notify-quantity-changes.ts

```diff
@@ -95,6 +95,18 @@ describe( 'notifyQuantityChanges', () => {
 		} );
 		expect( mockedCreateInfoNotice ).not.toHaveBeenCalled();
 	} );
+	it( 'does not show notices when the item is being added', () => {
+		const { oldCart, newCart } = getFreshCarts();
+		newCart.items[ 0 ].quantity = 5;
+		newCart.items[ 0 ].quantity_limits.maximum = 10;
```

Comment:
Given that we're not testing against the maximum allowed quantity limit, do we need this line here?

Reply:
Good point, done!